packages/form-integrations/test/helpers/umbrella-form.js

```diff
@@ -44,13 +44,13 @@ export class UmbrellaForm extends LitElement {
           <lion-input-date
             name="date"
             label="Date of application"
-            .modelValue="${new Date('2000-12-12')}"
+            .modelValue="${new Date('2000-12-12T00:00:00')}"
```

Comment:
Does this only occur in the tests, or do you also need to provide the time when you use it?

Reply:
I suppose giving the time is the only way to give the timezone. If you don't give the timezone, 

If I do a `new Date('2000-12-12')`, I have `Mon Dec 11 2000 19:00:00 GMT-0500 (Eastern Standard Time)`. The date selected is december 11.

Reply:
there is a lot of references to `new Date` in documentation, errors could happen for all people in a timezone with negative GMT time. We should maybe change the documentation about that?

Reply:
No, it's good in this code, so maybe it's something to be fixed with input-date?

![Capture](https://user-images.githubusercontent.com/9367152/85568736-f2062b80-b5ff-11ea-83fd-b82986032916.PNG)

Reply:
Yes, changing it in input-date sounds like the right approach.

Let me test your suggestion locally, to see if it works.

Reply:
I pushed a fix on the branch

Reply:
we should maybe have tests in a different timezone to avoid these issues. Chrome can be started with a different timezone:

https://gist.github.com/dustinsmith1024/dab3560abaa62cbe0754117d44c71026

Reply:
you just need to pass TZ environment variable to chrome. With karma, I think we need to start another script because karma use 1 browser for all tests.